slackbridge/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:wily
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Usual update / upgrade
+RUN apt-get update && apt-get upgrade -y && apt-get dist-upgrade -y
+
+# Install requirements
+RUN apt-get install -y curl supervisor
+
+# Install nodejs
+RUN curl -sL https://deb.nodesource.com/setup_5.x | bash -
+RUN apt-get install --fix-missing -y nodejs
+
+# Get slack-irc
+RUN npm install -g slack-irc
+
+# Add configurations
+ADD config.json slack-irc/config.json
```

Comment:
I would suggest to use COPY over ADD. The docker best practices usually prefer COPY. Possible suggestion, instead of COPY the configuration, maybe using VOLUME to pass it in. This way we can build one flexible image and all the user will have to do is pass the config via `docker run -v path:/slack-irc/`. This would avoid us having to rebuild the image every time.